### ISSUE TITLE
Provide an easier way to export and send log to us

### ISSUE THREAD

**Issue by ghost** [getlantern/lantern#4349]
I have many posts on the Chinese forum don't have log and I have to instruct to get it manually.

**Comment by atavism** [comment]
+1. I've wanted to add this to mobile for a long time now. It would definitely be useful for this first Pro release. We could add a "Send logs" side-menu option for it. Additionally, we could add a "profile" screen too, where a user could edit + verify her phone number, e-mail, and any other account details. That would go a long way towards providing more prioritized support for Pro, as you mentioned @percyalpha. What do you think, @Derekf5?

**Comment by Derekf5** [comment]
This sounds good to me  @percyalpha @atavism.  we already have the "pro account management" screen which we could essentially make into a "profile screen"  for Pro users.    Would we just want this to be a pro feature, or a feature for everyone.   What do you think @myleshorton?

**Comment by atavism** [comment]
Yeah, it would be really useful to have a place to go to to see my current phone number, to be able to change it, and to add/edit an e-mail address as well so I think the "pro account management" screen works well for that. @myleshorton I'm guessing it's too late (or not totally a necessity) for us to get this + e-mail address verification into this first release though, huh? Just thinking though how useful this would be for support.

**Comment by sunshine19090** [comment]
@Derekf5 @myleshorton I see that this has been added in mobile and it works. Found it super easy to send the logs as a user. It will be great if we can add this feature for desktop too. IMO, we should probably have this for free user too as we can spend less time describing on how to send it to us. It will be more helpful for us than the user I think.

**Comment by p3rcya** [comment]
@Derekf5 @myleshorton Definitely. And this should replace the contact us menu. We can change the wording "Send logs" to "report problems". Users can attach messages to the logs as well. Then we remove the contact us.

**Comment by Derekf5** [comment]
CC @atavism
